parm/atm/obs/config/satwind.yaml

```diff
@@ -13,226 +13,872 @@ obs space:
   simulated variables: [eastward_wind, northward_wind]
 get values:
   interpolation method: $(INTERP_METHOD)
-#
 obs operator:
   name: VertInterp
-#
-obs filters:
+obs pre filters:
+# Assign the initial observation error, based on height/pressure
+# Hard-wiring to prepobs_errtable.global by Type
+# ObsError is currently not updating in diag file, but passes directly to EffectiveError when no inflation is specified in YAML
+  # Type 240 (GOES SWIR): Assigned all dummy values in prepobs_errtable.global
+  - filter: Perform Action
+    filter variables:
+    - name: eastward_wind
+    - name: northward_wind
+    where:
+    - variable: ObsType/eastward_wind
+      is_in: 240
+    minvalue: -135.
+    maxvalue: 135.
+    action:
+      name: assign error
+      error function:
+        name: ObsErrorModelStepwiseLinear@ObsFunction
+        options:
+          xvar:
+            name: MetaData/air_pressure
+          xvals: [110000.,0.]   #Pressure (Pa)
+          errors: [1000000000.,1000000000.]
```

Comment:
If it is constant, why not just set it to one value everywhere? 

replace `error function: ...` with `error parameter: 1000000000.`. Should reduce YAML length.